.github/workflows/update-golint-version.yml

```diff
@@ -22,7 +22,9 @@ jobs:
         id: bumpGolint
         run: |
           make update-golint-version
-          echo "changes=$(git status --porcelain)" >> $GITHUB_OUTPUT
+          echo "changes<<EOF" >> $GITHUB_OUTPUT
+          echo "$(git status --porcelain)" >> $GITHUB_OUTPUT
+          echo "EOF" >> $GITHUB_OUTPUT
```

Comment:
I wonder if we can add a comment here for readabilty, why we are doing this ? (you could link to an issue or something